cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -178,8 +368,651 @@ class GameWorldService {
+                spawn_params.ObjectFlags = static_cast<EObjectFlags>(sp_spawn_params.getObj().ObjectFlags);
+
+                return reinterpret_cast<uint64_t>(
+                    UnrealClassRegistrar::spawnActor(
+                        class_name,
+                        world_,
+                        FVector(location.at(0), location.at(1), location.at(2)),
+                        FRotator(rotation.at(0), rotation.at(1), rotation.at(2)),
+                        spawn_params));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "create_component_outside_owner_constructor", 
+            [this](const std::string& class_name, const uint64_t& owner, const std::string& name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(UnrealClassRegistrar::createComponentOutsideOwnerConstructor(class_name, reinterpret_cast<AActor*>(owner), name));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "create_scene_component_outside_owner_constructor_from_actor",
+            [this](const std::string& class_name, const uint64_t& actor, const std::string& name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(UnrealClassRegistrar::createSceneComponentOutsideOwnerConstructor(class_name, reinterpret_cast<AActor*>(actor), name));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "create_scene_component_outside_owner_constructor_from_object",
+            [this](const std::string& class_name, const uint64_t& owner, const uint64_t& parent, const std::string& name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(
+                    UnrealClassRegistrar::createSceneComponentOutsideOwnerConstructor(class_name, reinterpret_cast<UObject*>(owner), reinterpret_cast<USceneComponent*>(parent), name));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "create_scene_component_outside_owner_constructor_from_component",
+            [this](const std::string& class_name, const uint64_t& owner, const std::string& name) -> uint64_t {
+                return reinterpret_cast<uint64_t>(UnrealClassRegistrar::createSceneComponentOutsideOwnerConstructor(class_name, reinterpret_cast<USceneComponent*>(owner), name));
+            });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "new_object",
+            [this](
+                const std::string& class_name,
+                const uint64_t& outer,
+                const std::string& name,
+                const std::map<std::string, std::string>& unreal_obj_strings,
+                const uint64_t& uobject_template,
+                const bool& copy_transients_from_class_defaults,
+                const uint64_t& in_instance_graph,
+                const uint64_t& external_package) -> uint64_t {
+
+                // TODO: support users to specify multiple object flags
```

Comment:
Should this say `UnrealObj<FSpObjectFlags>`?

```
// TODO: support multiple object flags
UnrealObj<FSpObjectFlags> sp_object_flags_obj("ObjectFlags");
UnrealObjUtils::setObjectPropertiesFromStrings({&sp_object_flags_obj}, unreal_obj_strings);

ESpObjectFlags sp_object_flags = sp_object_flags.getObj();
EObjectFlags object_flags = static_cast<EObjectFlags>(sp_object_flags.Enum);
```

Reply:
Yes, corrected this typo.